assets/src/blocks/Gallery/GalleryEditor.js

```diff
@@ -8,202 +8,201 @@ import { GalleryGrid } from './GalleryGrid';
+    <section className={ `block ${ GALLERY_BLOCK_CLASSES[ layout ] } ${ className ?? '' }` }>
       <header className="articles-title-container">
         <RichText
           tagName="h2"
           className="page-section-header"
-          placeholder={__('Enter title', 'planet4-blocks-backend')}
-          value={gallery_block_title}
-          onChange={toAttribute('gallery_block_title')}
+          placeholder={ __( 'Enter title', 'planet4-blocks-backend' ) }
+          value={ gallery_block_title }
+          onChange={ toAttribute( 'gallery_block_title' ) }
           withoutInteractiveFormatting
           multiline="false"
-          allowedFormats={[]}
+          allowedFormats={ [] }
         />
       </header>
       <RichText
         tagName="p"
         className="page-section-description"
-        placeholder={__('Enter description', 'planet4-blocks-backend')}
-        value={gallery_block_description}
-        onChange={toAttribute('gallery_block_description')}
+        placeholder={ __( 'Enter description', 'planet4-blocks-backend' ) }
+        value={ gallery_block_description }
+        onChange={ toAttribute( 'gallery_block_description' ) }
         withoutInteractiveFormatting
-        allowedFormats={['core/bold', 'core/italic']}
+        allowedFormats={ [ 'core/bold', 'core/italic' ] }
       />
-      {layout === 'slider' && <GalleryCarousel images={images || []} isEditing />}
-      {layout === 'three-columns' && <GalleryThreeColumns images={images || []} postType={postType} />}
-      {layout === 'grid' && <GalleryGrid images={images || []} />}
+      { layout === 'slider' && <GalleryCarousel images={ images || [] } isEditing /> }
+      { layout === 'three-columns' && <GalleryThreeColumns images={ images || [] } postType={ postType } /> }
+      { layout === 'grid' && <GalleryGrid images={ images || [] } /> }
     </section>
   );
-}
+};
 
-export const GalleryEditor = ({ isSelected, attributes, setAttributes }) => {
+export const GalleryEditor = ( { isSelected, attributes, setAttributes } ) => {
   const { multiple_image } = attributes;
 
-  const { image_urls_array } = useSelect(select => {
-    let image_urls_array = [];
-    if (multiple_image) {
-      let image_id_array = multiple_image.split(',');
-      image_id_array.forEach(img_id => {
-        let img_details = select('core').getMedia(img_id);
-        if (img_details) {
-          image_urls_array[img_id] = img_details.source_url;
+  const { image_urls_array } = useSelect( ( select ) => {
+    const image_urls_array_2 = [];
+    if ( multiple_image ) {
+      const image_id_array = multiple_image.split( ',' );
+      image_id_array.forEach( ( img_id ) => {
+        const img_details = select( 'core' ).getMedia( img_id );
+        if ( img_details ) {
+          image_urls_array_2[ img_id ] = img_details.source_url;
         }
-      });
+      } );
     }
-    return { image_urls_array };
-  }, []);
+    return { image_urls_array_2 };
```

Comment:
Here it should be `return { image_urls_array: image_urls_array_2 };` (also maybe we could rename it `image_urls_array_new` instead of `image_urls_array_2`? seems more accurate 🤔)

Reply:
Done